test/Frontend/crash-in-user-code.swift

```diff
@@ -0,0 +1,15 @@
+
+// RUN: echo %s > %t.filelist.txt
+// RUN: not --crash %target-swift-frontend -interpret -filelist %t.filelist.txt 2>&1 | %FileCheck %s
+
+// CHECK: Stack dump:
+// CHECK-NEXT: Program arguments:
+// CHECK-NEXT: Swift version
+// CHECK-NEXT: Contents of {{.*}}.filelist.txt:
+// CHECK-NEXT: ---
+// CHECK-NEXT: crash-in-user-code.swift
+// CHECK-NEXT: ---
+// CHECK-NEXT: While running user code "SOURCE_DIR/test/Front{{e|E}}nd/crash-in-user-code.swift"
```

Comment:
This is gonna fail on Windows, what if we just sidestepped the issue and matched
```suggestion
// CHECK-NEXT: While running user code "{{.*}}crash-in-user-code.swift"
```